controller/src/beerocks/cli/beerocks_cli_bml.h

```diff
@@ -67,11 +67,11 @@ class cli_bml : public cli {
     cli_bml(std::string beerocks_conf_path_);
     virtual ~cli_bml();
 
-    bool connect();
-    void disconnect();
-    bool is_connected();
-    bool start() { return true; };
-    void stop(){};
+    bool connect() override;
```

Comment:
Since override only applies to virtual function, I'd vote to drop it as well, it doesn't add any useful information.
If/when we introduce clang-tidy, there is even an option to do that automatically: https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-override.html